benchmarks/viscoelastic_plate_flexure/viscoelastic_plate_flexure.prm

```diff
@@ -0,0 +1,179 @@
+# This benchmark examines flexure of a finite-length elastic plate based
+# on the setup described in Choi et al., 2013, JGR, v.118, p.2429-2444,
+# doi:10.1002/jgrb.50148. The model setup has a 50 km long and 5 km thick 
+# elastic plate (shear modulus: 30 GPa, viscosity: 1e35 Pa s) overlaying a 
+# viscous layer (viscosity: 1e17 Pa s, 12.5 km thick), which both have a density
+# of 2700 kg/m^3. The top of the model is a free surface, while the bottom
+# boundary is zero-slip and the sides are free-slip. The upper right corner 
+# of the viscous layer is replaced with a low density (1890 kg/m^3) elastic 
+# block, which drives flexure of the elastic plate (right side goes up, left 
+# side goes down). The total topography obtained after the model has reached 
+# equilibrium is 330 m, with -146.5 m and 183.5 m of topography, respectively, 
+# on the left and right sides. The summed value and profiles differ slightly 
+# from those reported in Choi et al. (see viscoelastic_plate_flexure.png in 
+# this folder), but closely match more recent experiments performed by Cedric 
+# Thieulot. The curves from the experiments performed by Cedric Thieulot are 
+# labeled 'ELEFANT, nelx=150', 'ELEFANT, nelx=250', and '100x35'. For further 
+# details on these additional experiments, visit:
+# https://github.com/naliboff/fieldstone/blob/master/python_codes/fieldstone_64/results/flexureplate
+# The figure in this folder (viscoelastic_plate_flexure.png) was made and 
```

Comment:
Thanks for including the figure. Could you place the figure into `/benchmarks/viscoelastic_plate_flexure/doc/` and adjust the path in this text? That would follow the new convention for cookbook and benchmark figures.

Reply:
:+1: the figure is now in the `doc/` subfolder and the documentation is updated.